src/components/Recipe.js

```diff
@@ -0,0 +1,93 @@
+import React, { useEffect, useState } from "react";
+import { Link, useParams, useHistory } from "react-router-dom";
+import {
+  Title,
+  Button,
+  CircleList,
+  CircleListItem,
+  Container,
+} from "./Styles.js";
+import api from "../data/DataAPI.js";
+
+function IngredientList(props) {
+  const { ingredients } = props;
+  return (
+    <CircleList>
+      {ingredients &&
+        ingredients.map((ingredient) => (
+          <CircleListItem key={ingredient.name}>
+            {ingredient.name}
+          </CircleListItem>
+        ))}
+    </CircleList>
+  );
+}
+
+function RecipeDetail({ recipe }) {
+  const history = useHistory();
+
+  const deleteRecipe = () => {
```

Comment:
it would be nice to move all the api methods outside the component to an external module

Reply:
Moved all api methods to DataAPI.js